### ISSUE TITLE
Running MPRAflow without internet

### ISSUE THREAD

**Issue by xunchen85** [shendurelab/MPRAflow#46]
Hi,

Our HPC computing nodes do not have the access to the internet. Is there any way to run it without connection?

I am also trying it locally in my personal Ubuntu, although i got the error of "create BWA ref." which was previously reported to be the out of memory issue?

I am wondering what is the memory needed to run your test dataset?

Thanks,
Xun

**Comment by visze** [comment]
Hi Xun,

The issue #45 is not an memory issue. For the test set not much memory is needed. Here it seems that the design file is empty and this causes the error. Mabye it is the same in you case. can you check the `design_rmIllegalChars.fa` file in your out folder? Also the initial `design.fa` file if any of them is empty? On my side I cannot reproduce the error.

About your problem with internet connection. We are using conda to dowload the dependencies. So you need at least one node (maybe your login node) that has access to the internet and to you file system. There you can run mpraflow first. It should install the dependencies. Then you cancel it and run it on the cluster again using the `--resume` in nextflow.

Meanwhiel I will try to find the issue about the `create_BWA_ref` process..

Best,
Max

**Comment by xunchen85** [comment]
Hi Max,

I tried it on our login node, although our server set a cap of 8Gb. I was not able to run the association.nf tool with the Out of Memory issue even though MPRAflow may not need so much.

Regarding the create reference issue, sorry that I refer to your previous reply to #45. I have pasted some example lines of design.fa I obtained in #45, and the design_rmIllegalChars.fa is empty. 

Thanks,
Xun
